src/containerapp/azext_containerapp/_sdk_enums.py

```diff
@@ -335,6 +403,76 @@ class ManagedServiceIdentityType(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     SYSTEM_ASSIGNED_USER_ASSIGNED = "SystemAssigned,UserAssigned"
 
 
+class PatchApplyStatus(str, Enum, metaclass=CaseInsensitiveEnumMeta):
+    """The status of the patch once it has been provisioned."""
+
+    NOT_STARTED = "NotStarted"
+    REBASE_IN_PROGRESS = "RebaseInProgress"
+    CREATING_REVISION = "CreatingRevision"
+    SUCCEEDED = "Succeeded"
+    CANCELED = "Canceled"
+    REBASE_FAILED = "RebaseFailed"
+    REVISION_CREATION_FAILED = "RevisionCreationFailed"
+    IMAGE_PUSH_PULL_FAILED = "ImagePushPullFailed"
+    MANUALLY_SKIPPED = "ManuallySkipped"
+
+
+class PatchingMode(str, Enum, metaclass=CaseInsensitiveEnumMeta):
```

Comment:
I think we do not want to expose patch related property to customer? can we remove it?

Reply:
OK, I will remove them.